hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -266,8 +266,6 @@ public static Builder newBuilder() {
     auditMap.put(OzoneConsts.STORAGE_TYPE,
         (this.storageType != null) ? this.storageType.name() : null);
     auditMap.put(OzoneConsts.CREATION_TIME, String.valueOf(this.creationTime));
-    auditMap.put(OzoneConsts.OBJECT_ID, String.valueOf(this.getObjectID()));
-    auditMap.put(OzoneConsts.UPDATE_ID, String.valueOf(this.getUpdateID()));
```

Comment:
I am in 2 minds about this change. This is useful if someone deletes and re-creates a bucket with same name it would be easy for us to identify that information. No need to fix in this, just curious why you thought this is a bad idea.

Reply:
Does this break the Audit Parser code or Audit tests ?

Reply:
@bharatviswa504  pointed out that audit logs are actually used for finding out what operations were performed on the cluster, as was discussed in  some PR. And for debugging, we can use the debug logs if required. 
I am ok to go either way.